python/ucxx/_lib_async/notifier_thread.py

```diff
@@ -58,4 +58,7 @@ def _notifierThread(event_loop, worker, q):
         try:
             task.result(0.01)
         except TimeoutError:
+            task.cancel()
```

Comment:
**question:** Can this cancellation raise an error, and do we need to catch that too, or is there nothing we can do in that circumstance?

Reply:
My understanding is that when scheduling from another thread, `cancel()` will not raise as that needs to be `awaited`, see how the `[asyncio.run_coroutin_threadsafe()` docs](https://docs.python.org/3/library/asyncio-task.html#task-cancellation) have an almost verbatim example.